CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the create-aptos-dapp tool will be captured in this file.
 
 # Unreleased
 
+- Update indexer template's indexer sdk dependency
```

Comment:
can we mention the dependency version we bump to?

Reply:
We are still using commit hash now, indexer team hasn't tagged a version yet